### ISSUE TITLE
ICN does not detect `networkx as nx`

### ISSUE THREAD

**Issue by AbdealiLoKo** [astral-sh/ruff#6763]
I was using the `flake8-import-conventions`'s `ICN` rules in my project.
I notice that the `networkx as nx` is not implemented in ruff's [settings](https://github.com/astral-sh/ruff/blob/0f9ccfcad9f00e820871fd01e6c3c234cf87ac7b/crates/ruff/src/rules/flake8_import_conventions/settings.rs#L8)

The original library has support for `networkx` in ICN004 - https://github.com/joaopalmeiro/flake8-import-conventions#flake8-codes

TO reproduce:
```
$ cat /tmp/try.py
import networkx

$ ruff /tmp/try.py --isolated --select ICN
$ echo $?
0

$ ruff --version
ruff 0.0.285
```

**Comment by charliermarsh** [comment]
This may've just been an oversight. PR welcome if you'd like :)

**Comment by charliermarsh** [comment]
This is also configurable: https://beta.ruff.rs/docs/settings/#flake8-import-conventions-aliases.
